### PR TITLE
update QuickLRU option (2)

### LINKED ISSUE #14: LRU max age
```ts
export interface LRUOptions {
  readonly maxAge?: number
  readonly maxSize: number
}
```

```ts
this.memoizedResults = new QuickLRU({
      maxSize: this.options?.lru?.maxSize ? this.options.lru.maxSize : 100,
})
```

Thanks for you work, and I recently noticed that you provide LRUOptions with maxAge, but did not pass to QuickLRU.

### PR DESCRIPTION
**Pull request by shkaper**
Following up on #15, I've fixed the flaking test there. I think sinonjs FakeTimer is a more robust solution for a test where timeouts are involved.

Closes #14

### CODE REVIEW

File: test/http-data-source.test.ts [hunk 1]
```diff
@@ -1336,12 +1337,15 @@ test('Should only cache GET successful responses with the correct status code',
   t.is(cacheMap.size, 0)
 })
 
-test('Global maxAge should be used when no maxAge was set or similar.', async (t) => {
+test.serial('Global maxAge should be used when no maxAge was set or similar.', async (t) => {
```
Comment (shkaper) [inline test/http-data-source.test.ts:1340]: using `serial()` here because otherwise FakeTimers mess up the timeouts in concurrent tests

**Comment by shkaper** [comment]
Hey @StarpTech, have you got the chance to review this yet?

File: src/http-data-source.ts [hunk 2]
```diff
@@ -93,6 +93,7 @@ export abstract class HTTPDataSource<TContext = any> extends DataSource {
     super()
     this.memoizedResults = new QuickLRU({
       maxSize: this.options?.lru?.maxSize ? this.options.lru.maxSize : 100,
+      maxAge: this.options?.lru?.maxAge,
```
Comment (StarpTech) [inline src/http-data-source.ts:96]: Apollo works with seconds. Should we work with seconds too here?

**Review by StarpTech — CHANGES_REQUESTED** [review]

**Comment by thiagomini** [comment]
Hi @shkaper and @StarpTech , we are in need of this feature as well. Cna you merge this and update the package?

**Comment by StarpTech** [comment]
Hi @thiagomini, the PR is reviewed, but the original author isn't responding. In the future, I recommend creating a separate PR. I'll handle it for now.

**Pull request merged**